2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,39 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+const recipes = [
+  {
+    title: "Lasagna",
+    serves: 6,
+    ingredients: ["dough", "cheese", "pepperoni"],
+  },
+  {
+    title: "Soup",
+    serves: 4,
+    ingredients: ["leeks", "broccoli", "blue cheese"],
+  },
+  {
+    title: "Tuna Pasta",
+    serves: 2,
+    ingredients: ["penne", "tuna", "onions", "olives"],
+  },
+  {
+    title: "Risotto",
+    serves: 4,
+    ingredients: ["rice", "parmesan", "sundried tomatoes"],
+  },
+  {
+    title: "Mole",
+    serves: 2,
+    ingredients: ["cinnamon", "cumin", "cocoa"],
+  },
+];
+
+recipes.forEach((recipe) => {
```

Comment:
this is really good! one tip is that so you don't need to keep writing `recipe.something`, `recipe.somethingElse` you can do what is call destructuring the recipe at the start ->

` recipes.forEach(({title, serves, ingredients}) => {
  console.log(title);
  console.log('Serves: ${serves}');
  console.log("Ingredients:");
 ingredients.forEach((ingredient) => {
    console.log(ingredient);
  });
});`